### PR TITLE
add div by zero checks to creative tanks and chests

### PR DESCRIPTION
**Pull request by htmlcsjs**

### CODE REVIEW

**Comment by ALongStringOfNumbers** [comment]
Instead just change the validation on the ticks per cycle widget to not accept 0?

**Comment by htmlcsjs** [comment]
ill do that later, this was a quick fix to unbrick my world

File: src/main/java/gregtech/common/metatileentities/storage/MetaTileEntityCreativeChest.java [hunk 1]
```diff
@@ -98,7 +98,7 @@ protected ModularUI createUI(EntityPlayer entityPlayer) {
     @Override
     public void update() {
         super.update();
-        if (getOffsetTimer() % ticksPerCycle != 0) return;
+        if (ticksPerCycle != 0 || getOffsetTimer() % ticksPerCycle != 0) return;
```
Comment (ALongStringOfNumbers) [inline src/main/java/gregtech/common/metatileentities/storage/MetaTileEntityCreativeChest.java:101]: Shouldn't this be `ticksPerCycle == 0`? Same for the Creative tank

Comment (htmlcsjs) [reply src/main/java/gregtech/common/metatileentities/storage/MetaTileEntityCreativeChest.java]: fixed

**Review by brachy84 — APPROVED** [review]

**Pull request merged**